<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newAppointment</class>
 <widget class="QWidget" name="newAppointment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Btn_yes">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>确认预约</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>就诊时段</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_time">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>上午</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>下午</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>晚上</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_doc_gender_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>医生性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_doc_name_title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>医生</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_doc_name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>赵奶砖</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_user_title">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>就诊人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_doc_gender">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_cancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_doc_apartment_title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>科室</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_doc_apartment">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>牛马科</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>就诊日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>11/45/14</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_user">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>小明</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哈哈哈</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Btn_add_user">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>添加就诊人</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
